drivers/spi/spi_ll_stm32.c

```diff
@@ -683,6 +782,19 @@ static int transceive(const struct device *dev,
 		spi_context_buffers_setup(&data->ctx, tx_bufs, rx_bufs, 2);
 	}
 
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+	if (SPI_OP_MODE_GET(config->operation) == SPI_OP_MODE_MASTER) {
```

Comment:
I guess this only relates to fifo-enabled operation.
Why not doing it under fifo-enabled condition ?

Reply:
Absolutely, this should happen only if FIFO is enabled.

Reply:
Yes, it should be under FIFO-enabled operation.